src/app.postcss

```diff
@@ -1,8 +1,8 @@
 @import "highlight.js/scss/github-dark.scss";
 
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
```

Comment:
Why is this like this?

Reply:
I am not a big fan of this approach tbh. Lets see if we can revert it

Reply:
> I am not a big fan of this approach tbh. Lets see if we can revert it

Okay sure I will take a look, yea I think this isnt happening with other routes, anyways I dont like this approach either :\

Reply:
> Okay sure I will take a look, yea I think this isnt happening with other routes, anyways I dont like this approach either :\

oh :\ you should have notified me this